common/list/lists-en.html

```diff
@@ -0,0 +1,583 @@
+	<li>Item 3</li>
+	<li>Item 4</li>
+	<li>Item 5</li>
+	<li>Item 6</li>
+	<li>Item 7</li>
+	<li>Item 8</li>
+</ul>
+<pre>
+	<code>&lt;ul <strong>class=&quot;colcount-sm-3&quot;</strong>&gt;
+	&lt;li&gt;Item 1&lt;/li&gt;
+	...
+	&lt;li&gt;Item 8&lt;/li&gt;
+	&lt;/ul&gt;</code></pre>
+<h5> Appearance − Four columns</h5>
+<ul class="colcount-sm-4">
+	<li>Item 1</li>
+	<li>Item 2</li>
+	<li>Item 3</li>
+	<li>Item 4</li>
+	<li>Item 5</li>
+	<li>Item 6</li>
+	<li>Item 7</li>
+	<li>Item 8</li>
+</ul>
+<pre>
+	<code>&lt;ul <strong>class=&quot;colcount-sm-4&quot;</strong>&gt;
+	&lt;li&gt;Item 1&lt;/li&gt;
+	...
+	&lt;li&gt;Item 8&lt;/li&gt;
+	&lt;/ul&gt;</code></pre>
+<h5> Appearance − Multi-class</h5>
+<ul class="colcount-sm-2 colcount-md-3 colcount-lg-4">
+	<li>Item 1</li>
+	<li>Item 2</li>
+	<li>Item 3</li>
+	<li>Item 4</li>
+	<li>Item 5</li>
+	<li>Item 6</li>
+	<li>Item 7</li>
+	<li>Item 8</li>
+</ul>
+<pre>
+	<code>&lt;ul <strong>class=&quot;colcount-sm-2 colcount-md-3 colcount-lg-4&quot;</strong>&gt;
+	&lt;li&gt;Item 1&lt;/li&gt;
+	...
+	&lt;li&gt;Item 8&lt;/li&gt;
+	&lt;/ul&gt;</code></pre>
+
+<h4> Responsive list".list-responsive" </h4>
+<ul class="wb-eqht list-unstyled mrgn-tp-md mrgn-bttm-sm lst-spcd-2 list-responsive wb-init wb-eqht-inited" id="wb-auto-5">
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+</ul>
+<h3>Source code</h3>
+<pre>
+	<code>&lt;ul class="<strong>list-responsive</strong>"&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;/ul&gt;
+	</code></pre>
+<h4 id="wb-cont" property="name">List - Additional styles - GCWeb theme V5</h4>
+<h5>List double spaced <code>.lst-spcd-2</code></h5>
+<ul class="lst-spcd-2">
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+</ul>
+<h3>Source code</h3>
+<pre>
+	<code>&lt;ul class="lst-spcd-2"&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;/ul&gt;</code></pre>
+<h5>Lists in columns <code>.list-col-[breakpoint]-[columns]</code></h5>
+<ul class="list-col-xs-1 list-col-sm-2 list-col-md-3 list-col-lg-4">
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+</ul>
+<h3>Source code</h3>
+<pre>
+	<code>&lt;ul class="<strong>list-col-xs-1 list-col-sm-2 list-col-md-3 list-col-lg-4</strong>"&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;/ul&gt;
+	</code></pre>
+
+<h1 id="wb-cont" property="name">List group</h1>
```

Comment:
Please have only one heading level 1 per page or you can split the content onto multiple pages.